docs/collector.hyperv.md

```diff
@@ -96,7 +96,18 @@ Name | Description | Type | Labels
 _This collector does not yet have explained examples, we would appreciate your help adding them!_
 
 ## Useful queries
-_This collector does not yet have any useful queries added, we would appreciate your help adding them!_
+Percent of physical CPU ressources used per VM (on instance "localhost")
+```
+(sum (rate(windows_hyperv_vm_cpu_hypervisor_run_time{instance="localhost"}[1m]))) / ignoring(vm) group_left max (windows_cs_logical_processors{instance="localhost"}) / 100000
```

Comment:
This "magic constant" 100000, do we know why it is required? Could it be a bug in the hyperv collector?

Reply:
It's probably related to the time format. Some king of Hypervisor Tick... I already saw 1000 or 10000 on some places, but 100000 never.
Poor documentation on those from MS :)

Reply:
There's a factor 10,000,000 that occurs in a couple of similar places for ticks-to-seconds conversions. But let's leave that out of scope for this PR :)